app/konnect/updates.md

```diff
@@ -11,6 +11,13 @@ services. [Try it today!](https://cloud.konghq.com/quick-start)
 
 ## October 2022
 
+
+### 2022.10.24
```

Comment:
This will be a November release, October was only internal. Can coordinate w/ Smriti on release date .